### PR TITLE
Update from talk template

### PR DESCRIPTION
**Pull request by berziiii**
Remove unnecessary files
Provide starter content to README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,271 +1,266 @@
-[![General Assembly Logo](https://camo.githubusercontent.com/1a91b05b8f4d44b5bbfb83abac2b0996d8e26c92/687474703a2f2f692e696d6775722e636f6d2f6b6538555354712e706e67)](https://generalassemb.ly/education/web-development-immersive)
+![General Assembly Logo](http://i.imgur.com/ke8USTq.png)
```
Comment (jrhorn424) [inline README.md]: This is wrong.

File: README.md [hunk 2]
```diff
@@ -1,271 +1,266 @@
...
+# Rails API Single Resource
 
-A template for starting projects with `rails-api`. Includes authentication.
+## Objectives
 
-At the beginning of each cohort, update the versions in [`Gemfile`](Gemfile).
+By the end of this lesson, students should be able to:
 
-## Dependencies
+-   ?
```
Comment (jrhorn424) [inline README.md:9]: This was an intentional marker, yes?

File: README.md [hunk 3]
```diff
@@ -1,271 +1,266 @@
...
+## Status Check
 
-Response:
+These will be frequent. As developers we want to be meticulous and make sure
+we're getting errors where expected as we build our API.
 
-```md
-HTTP/1.1 201 Created
-Content-Type: application/json; charset=utf-8
+## Demo: [Library-Demo app](https://github.com/ga-wdi-boston/rails-api-library-demo)
```
Comment (jrhorn424) [inline README.md]: Please change to `[Library App]`

File: README.md [hunk 4]
```diff
@@ -1,271 +1,266 @@
...
-  "user": {
-    "id": 1,
-    "email": "an@example.email"
-  }
-}
-```
+## Code-Along: [Clinic-code-along-app](https://github.com/ga-wdi-boston/rails-api-clinic-code-along)
 
-#### POST /sign-in
+## Lab: [Cookbook-lab-app](https://github.com/ga-wdi-boston/rails-api-cookbook-lab)
```
Comment (jrhorn424) [inline README.md]: [Cookbook App]

File: README.md [hunk 5]
```diff
@@ -1,271 +1,266 @@
...
+-   **[Official Rails Documentation](http://rubyonrails.org/documentation/)**
+-   **[MVC](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller)**
+-   **[JSON API](https://thesocietea.org/2015/02/building-a-json-api-with-rails-part-1-getting-started/)**
 
 ## [License](LICENSE)
 
 1.  All content is licensed under a CC­BY­NC­SA 4.0 license.
-1.  All software code is licensed under GNU GPLv3. For commercial use or
-    alternative licensing, please contact legal@ga.co.
+1.  All software code is licensed under GNU GPLv3. For commercial use or alternative licensing, please contact legal@ga.co.
```
Comment (jrhorn424) [inline README.md]: This change violates the linter.

**Review by jrhorn424 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (payne-chris-r) [reply README.md:9]: Yes

File: README.md (hunk 5 above)
Comment (payne-chris-r) [reply README.md]: You violate the linter!

**Pull request merged**